### PR TITLE
josm: init at 9060

### PR DESCRIPTION
**Pull request by rycee**

### CODE REVIEW

File: pkgs/applications/misc/josm/default.nix [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+  phases = [ "installPhase" ];
+
+  buildInputs = [ jre8 ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cat > $out/bin/josm <<EOF
+    #!${bash}/bin/bash
+    exec ${jre8}/bin/java -jar $src "\$@"
```
Comment (the-kenny) [inline pkgs/applications/misc/josm/default.nix]: I'm not sure if referencing `$src` from an installed file is very clean (makes it hard to find the jar, as you'd expect it in `$out`). Maybe copy the jar to `$out/share/josm`?

**Comment by the-kenny** [comment]
Else it looks good to me (and works fine)

File: pkgs/applications/misc/josm/default.nix (hunk 1 above)
Comment (rycee) [inline pkgs/applications/misc/josm/default.nix]: Fair point. I've force pushed an amended commit that copies it to `$out/share/java/josm.jar`.

**Comment by the-kenny** [comment]
Merged as d491005

**Pull request closed without merging**